src/model/UserphraseModel.cpp

```diff
@@ -157,9 +157,9 @@ void UserphraseModel::refresh()
     emit refreshCompleted(userphrase_.size());
 }
 
-void UserphraseModel::add(std::shared_ptr<QString> phrase, std::shared_ptr<QString> bopomofo)
+void UserphraseModel::addWrapper(QString &phrase, QString &bopomofo)
```

Comment:
@Chocobo1 :
I thought to keep the code structure and only modified the parameter type,
if you think the private `add()` can be moved to public, and I will do the change.